{"uuid": "00ab2421-63fe-4f9c-be08-18a3437566b4", "children": ["6792c617-b935-4b18-a2a3-44f7a43203da"], "befores": [{"name": "login", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/Users/dmytro.maksymov/PycharmProjects/self/venv/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/venv/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/venv/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/regular/tests/test_selenium_elemental.py\", line 8, in login\n    return login_page.LoginPage(driver)\n  File \"/Users/dmytro.maksymov/PycharmProjects/self/regular/pages/login_page.py\", line 16, in __init__\n    assert self._is_displayed(self._login_form)\n"}, "start": 1614289419012, "stop": 1614289419879}], "start": 1614289419012, "stop": 1614289419895}